Casks/w/webull.rb

```diff
@@ -1,19 +1,22 @@
 cask "webull" do
-  version "7.4.2"
-  sha256 "d61a52dda9afde41dd058dc955fcec037de62b568aabc9e8e82ccaf58f0cb77e"
+  version "7.11.5,70007144"
+  sha256 "1bde1e54b5a0f98569161f4a7f31a3d7ccc1ff84b06aef942d713726b6fa431f"
 
-  url "https://u1sweb.webullfintech.com/us/Webull%20Desktop_#{version}_global_universalsigned.dmg",
+  url "https://u1sweb.webullfintech.com/us/Webull%20Desktop_#{version.csv.first}_#{version.csv.second}_global_universalsigned.dmg",
       verified: "u1sweb.webullfintech.com/us/"
   name "Webull"
   desc "Desktop client for Webull Financial LLC"
   homepage "https://www.webull.com/"
 
   livecheck do
-    url "https://infoapi.webullfintech.com/api/operation/appver/last?platform=qt_mac_global&osv=10.14"
-    regex(/Webull%20Desktop[._-](\d+(?:\.\d+)+).*?.dmg/i)
+    url "https://infoapi.webullfintech.com/api/operation/appver/last?platform=qt_mac_global&osv=10.15"
+    regex(/Webull%20Desktop[._-](\d+(?:\.\d+)+(?:_\d+)).*?\.dmg/i)
+    strategy :json do |json, regex|
+      json["upgradeUrl"]&.scan(regex)&.map { |match| (match[0]).to_s.tr("_", ",") }
```

Comment:
```suggestion
    url "https://infoapi.webullfintech.com/api/operation/appver/last?platform=qt_mac_global&osv=14"
    regex(/Webull%20Desktop[._-]v?(\d+(?:[._]\d+)+).*?\.dmg/i)
    strategy :json do |json, regex|
      json["upgradeUrl"]&.scan(regex)&.map { |match| match[0].tr("_", ",") }
```

* Using the lowest supported macOS version may cause the check to always return an older version if/when upstream releases a new version with a higher minimum macOS requirement. If we use the current macOS version (14), that will help to keep this working for longer (though it can break in the same way over time).
* I've added some tweaks to the regex to bring it in line with similar regexes.
* We can omit `#to_s` from `match[0].to_s.tr("_", ",")`.